sos/report/plugins/charmed_mysql.py

```diff
@@ -29,6 +29,40 @@ class CharmedMySQL(Plugin, UbuntuPlugin):
     plugin_name = "charmed_mysql"
     packages = ("charmed-mysql",)
 
+    mysql_queries = [
+        # Get databases user has access to
+        "'show databases;'",
```

Comment:
I understand you just move this code around, so this is more nitpickyou can ignore.

Are the extra `'..'` quotation marks really needed? And if so, isnt it better to put them to the

    [f"{sql_cmd} {query}" for query in self.mysql_queries],

command list? I mean, maintaining this `mysql_queries` list can be error-prone with the extra quote marks..?

(but again, I leave on you to decide if it is worth adjusting this or not)